.github/CODEOWNERS

```diff
@@ -1,21 +1,6 @@
 # NB: Last matching rule takes precedence in CODEOWNERS.
 
-# Fall-through to community maintainers.
-* @thundergolfer
-
-# Core Python rules belong to the Bazel team.
-/python/ @brandjon @lberki
-# But not everything under python/ is the core Python rules.
-/python/pip.bzl @thundergolfer
-/python/requirements.txt @thundergolfer
+* @rickeylev
```

Comment:
Are you making yourself the sole reviewer? I'm okay with that but it seems like it might get overwhelming after a while.

Reply:
Oh yes, it's definitely going to be overwhelming. I'd love to have some more entries in here (Henry said he's willing to be added to cover pip_parse). This is just a minimal change to prevent every PR being auto-assigned to people who won't respond.